client/a8c-for-agencies/sections/reports/primary/build-report/index.tsx

```diff
@@ -0,0 +1,344 @@
+import {
+	Button,
+	SelectControl,
+	TextareaControl,
+	CheckboxControl,
+	TextControl,
+	DatePicker,
+	Popover,
+} from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import A4ASelectSite from 'calypso/a8c-for-agencies/components/a4a-select-site';
+import { LayoutWithGuidedTour as Layout } from 'calypso/a8c-for-agencies/components/layout/layout-with-guided-tour';
+import LayoutTop from 'calypso/a8c-for-agencies/components/layout/layout-with-payment-notification';
+import MobileSidebarNavigation from 'calypso/a8c-for-agencies/components/sidebar/mobile-sidebar-navigation';
+import LayoutBody from 'calypso/layout/hosting-dashboard/body';
+import LayoutHeader, {
+	LayoutHeaderBreadcrumb as Breadcrumb,
+	LayoutHeaderActions as Actions,
+} from 'calypso/layout/hosting-dashboard/header';
+import { A4A_REPORTS_LINK } from '../../constants';
+
+import './style.scss';
+
+const AVAILABLE_TIMEFRAMES = [
+	{ label: 'Last 30 days', value: '30_days' },
+	{ label: 'Last 7 days', value: '7_days' },
+	{ label: 'Last 24 hours', value: '24_hours' },
+	{ label: 'Custom range', value: 'custom' },
```

Comment:
Should these be translatable strings?

Reply:
Good catch. Fixed it here: https://github.com/Automattic/wp-calypso/pull/104080/commits/94ee9d307710a4de2eaff26a844cd4e9c351222b